pkg/kubelet/winstats/perfcounter_nodestats_unsupported.go

```diff
@@ -0,0 +1,26 @@
+//go:build !windows
+// +build !windows
+
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package winstats
+
+import "errors"
+
+func GetAvailableAndTotalPhysicalMemory() (uint64, uint64, error) {
```

Comment:
This function doesn't look to be used outside windows, I don't think this is required anymore (was maybe part of previous versions of the PR?)

Reply:
you are correct, let me remove it

Reply:
updated!